### ISSUE TITLE
Several blocks do not define the standard capability myaddinstance

### ISSUE THREAD

**Issue by rosa2** [iomad/iomad#343]
Hello

I have all debugging Moodle tools on and when I go to my dashboard (nor Iomad, the user one) I get:

```
The block iomad_commerce does not define the standard capability block/iomad_commerce:myaddinstance

    line 602 of /blocks/moodleblock.class.php: call to debugging()
    line 574 of /blocks/moodleblock.class.php: call to block_base->has_add_block_capability()
    line 218 of /lib/blocklib.php: call to block_base->user_can_addto()
    line 1949 of /lib/blocklib.php: call to block_manager->get_addable_blocks()
    line 1027 of /lib/blocklib.php: call to block_add_block_ui()
    line 335 of /lib/blocklib.php: call to block_manager->ensure_content_created()
    line 18 of /theme/intermedia/layout/default.php: call to block_manager->region_has_content()
    line 1018 of /lib/outputrenderers.php: call to include()
    line 948 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
    line 164 of /my/index.php: call to core_renderer->header()

The block iomad_company_admin does not define the standard capability block/iomad_company_admin:myaddinstance

    line 602 of /blocks/moodleblock.class.php: call to debugging()
    line 574 of /blocks/moodleblock.class.php: call to block_base->has_add_block_capability()
    line 218 of /lib/blocklib.php: call to block_base->user_can_addto()
    line 1949 of /lib/blocklib.php: call to block_manager->get_addable_blocks()
    line 1027 of /lib/blocklib.php: call to block_add_block_ui()
    line 335 of /lib/blocklib.php: call to block_manager->ensure_content_created()
    line 18 of /theme/intermedia/layout/default.php: call to block_manager->region_has_content()
    line 1018 of /lib/outputrenderers.php: call to include()
    line 948 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
    line 164 of /my/index.php: call to core_renderer->header()

The block iomad_company_selector does not define the standard capability block/iomad_company_selector:myaddinstance

    line 602 of /blocks/moodleblock.class.php: call to debugging()
    line 574 of /blocks/moodleblock.class.php: call to block_base->has_add_block_capability()
    line 218 of /lib/blocklib.php: call to block_base->user_can_addto()
    line 1949 of /lib/blocklib.php: call to block_manager->get_addable_blocks()
    line 1027 of /lib/blocklib.php: call to block_add_block_ui()
    line 335 of /lib/blocklib.php: call to block_manager->ensure_content_created()
    line 18 of /theme/intermedia/layout/default.php: call to block_manager->region_has_content()
    line 1018 of /lib/outputrenderers.php: call to include()
    line 948 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
    line 164 of /my/index.php: call to core_renderer->header()

The block iomad_link does not define the standard capability block/iomad_link:myaddinstance

    line 602 of /blocks/moodleblock.class.php: call to debugging()
    line 574 of /blocks/moodleblock.class.php: call to block_base->has_add_block_capability()
    line 218 of /lib/blocklib.php: call to block_base->user_can_addto()
    line 1949 of /lib/blocklib.php: call to block_manager->get_addable_blocks()
    line 1027 of /lib/blocklib.php: call to block_add_block_ui()
    line 335 of /lib/blocklib.php: call to block_manager->ensure_content_created()
    line 18 of /theme/intermedia/layout/default.php: call to block_manager->region_has_content()
    line 1018 of /lib/outputrenderers.php: call to include()
    line 948 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
    line 164 of /my/index.php: call to core_renderer->header()

The block iomad_onlineusers does not define the standard capability block/iomad_onlineusers:myaddinstance

    line 602 of /blocks/moodleblock.class.php: call to debugging()
    line 574 of /blocks/moodleblock.class.php: call to block_base->has_add_block_capability()
    line 218 of /lib/blocklib.php: call to block_base->user_can_addto()
    line 1949 of /lib/blocklib.php: call to block_manager->get_addable_blocks()
    line 1027 of /lib/blocklib.php: call to block_add_block_ui()
    line 335 of /lib/blocklib.php: call to block_manager->ensure_content_created()
    line 18 of /theme/intermedia/layout/default.php: call to block_manager->region_has_content()
    line 1018 of /lib/outputrenderers.php: call to include()
    line 948 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
    line 164 of /my/index.php: call to core_renderer->header()

The block iomad_reports does not define the standard capability block/iomad_reports:myaddinstance

    line 602 of /blocks/moodleblock.class.php: call to debugging()
    line 574 of /blocks/moodleblock.class.php: call to block_base->has_add_block_capability()
    line 218 of /lib/blocklib.php: call to block_base->user_can_addto()
    line 1949 of /lib/blocklib.php: call to block_manager->get_addable_blocks()
    line 1027 of /lib/blocklib.php: call to block_add_block_ui()
    line 335 of /lib/blocklib.php: call to block_manager->ensure_content_created()
    line 18 of /theme/intermedia/layout/default.php: call to block_manager->region_has_content()
    line 1018 of /lib/outputrenderers.php: call to include()
    line 948 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
    line 164 of /my/index.php: call to core_renderer->header()
```

**Comment by turf212** [comment]
Will need to get these added. Ta!

**Comment by turf212** [comment]
fixed these all in 3.0 version.
